src/ansys/geometry/core/designer/face.py

```diff
@@ -287,6 +299,67 @@ def loops(self) -> list[FaceLoop]:
 
         return loops
 
+    @property
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def color(self) -> str:
+        """Get the current color of the face."""
+        if self._color is None and self.body.is_alive:
+            # Assigning default value first
+            self._color = Color.DEFAULT.value
+
+            # If color is not cached, retrieve from the server
+            response = self._faces_stub.GetColor(EntityIdentifier(id=self.id))
+
+            # Return if valid color returned
+            if response.color:
+                self._color = mcolors.to_hex(response.color)
+            else:
+                self._color = Color.DEFAULT.value
+
+        return self._color
+
+    @color.setter
+    def color(self, color: str | tuple[float, float, float]) -> None:
+        self.set_color(color)
+
+    @protect_grpc
+    @check_input_types
+    @min_backend_version(25, 2, 0)
+    def set_color(self, color: str | tuple[float, float, float]) -> None:
+        """Set the color of the face."""
+        self._grpc_client.log.debug(f"Setting face color of {self.id} to {color}.")
+
+        try:
+            if isinstance(color, tuple):
+                # Ensure that all elements are within 0-1 or 0-255 range
+                if all(0 <= c <= 1 for c in color):
+                    # Ensure they are floats if in 0-1 range
+                    if not all(isinstance(c, float) for c in color):
+                        raise ValueError("RGB values in the 0-1 range must be floats.")
+                elif all(0 <= c <= 255 for c in color):
+                    # Ensure they are integers if in 0-255 range
+                    if not all(isinstance(c, int) for c in color):
+                        raise ValueError("RGB values in the 0-255 range must be integers.")
+                    # Normalize the 0-255 range to 0-1
+                    color = tuple(c / 255.0 for c in color)
+                else:
+                    raise ValueError("RGB tuple contains mixed ranges or invalid values.")
+
+                color = mcolors.to_hex(color)
+            elif isinstance(color, str):
+                color = mcolors.to_hex(color)
+        except ValueError as err:
+            raise ValueError(f"Invalid color value: {err}")
```

Comment:
Code duplication again. Maybe it's a good chance to move this method elsewhere (auxiliary function) and reuse it when needed.